video_recording_test.go

```diff
@@ -0,0 +1,30 @@
+package twilio
+
+import (
+	"context"
+	"testing"
+)
+
+func TestGetVideoRecording(t *testing.T) {
+	t.Parallel()
+	client, server := getServer(videoRecordingResponse)
+	defer server.Close()
+	recording, err := client.Video.VideoRecordings.Get(context.Background(), "RT63868a235fc1cf3987e6a2b67346273f")
```

Comment:
From an API perspective there's a little stutter here where we write Video twice. Would it be better as client.Video.Recordings.Get()?

(Or that overlap with the existing Recordings struct, huh. Ugh.)

Reply:
I saw that but I wasn't sure what to make with it. Also I find that when using `Client` it's easy to make `null pointer dereference`.
To solve that I think in this place in `Client` declaration I should remove `Rooms` and `VideoRecordings` and make some `VideoClient` struct with Client and this two services 
```
type VideoClient struct {
	*Client
	Rooms *RoomService
	Recordings *VideoRecordingService
}
``` 
Also I think it would be good for other services too, what you think?

Reply:
Maybe for a separate pull request?

Reply:
so I should leave this VideoClient as it is now, or you mean to move into seperate pull request changing all other Clients and here change only VideoClient?